src/services/devHandler/newsProcessDevProcess.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable camelcase */
+const {v4: uuidv4} = require('uuid');
+const {newsQueue} = require('../../config');
+const {checkIfValidURL, successResponse, errorResponse} = require('../../utils');
+
+const newsProcessDevProcess = async (req, res) => {
+  const bodyData = req.body?.message;
+  const id_feed = req.body?.id || false;
+  const user_id = req.body?.actor?.id || false;
+  const topics = req.body?.topics || [];
+  const duration_feed = req.body?.duration_feed || '';
+
+  const options = {
+    jobId: uuidv4(),
+    removeOnComplete: true
+  };
+
+  console.log('check if valid url', checkIfValidURL(bodyData));
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to leave console.log statements in production code as they can expose sensitive information and slow down your application. Consider using a logging library that can be toggled for development/production environments.

```diff
- 18:   console.log('check if valid url', checkIfValidURL(bodyData));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->